SoloPlay/scripts/mods/SoloPlay/SoloPlay_localization.lua

```diff
@@ -68,198 +71,247 @@ return {
 	button_start_normal = {
 		en = "Play Normal Game",
 		["zh-cn"] = "开始常规游戏",
+		ru = "Запуск обычной игры",
 	},
 	button_start_havoc = {
 		en = "Play Havoc Game",
 		["zh-cn"] = "开始浩劫游戏",
+		ru = "Запуск Верной смерти",
 	},
 	button_randomize = {
 		en = "Regenerate",
 		["zh-cn"] = "重新生成",
+		ru = "Перегенерировать",
 	},
 	button_modifier_customizable = {
 		en = "Customizable",
 		["zh-cn"] = "手动调整",
+		ru = "Настроить",
 	},
 	title_normal_mode = {
 		en = "Normal Mode",
 		["zh-cn"] = "常规模式",
+		ru = "Обычный режим",
 	},
 	title_havoc_mode = {
 		en = "Havoc Mode",
 		["zh-cn"] = "浩劫模式",
+		ru = "Режим «Верной смерти»",
 	},
 	label_mission = {
 		en = "Mission",
 		["zh-cn"] = "任务",
+		ru = "Миссия",
 	},
 	label_side_mission = {
 		en = "Side Mission",
 		["zh-cn"] = "次要目标",
+		ru = "Дополнительная миссия",
 	},
 	label_circumstance = {
 		en = "Special Condition",
 		["zh-cn"] = "特殊状况",
+		ru = "Особое условие",
 	},
 	label_circumstance1 = {
 		en = "Havoc Condition 1",
 		["zh-cn"] = "浩劫状况 1",
+		ru = "Условие Верной смерти 1",
 	},
 	label_circumstance2 = {
 		en = "Havoc Condition 2",
 		["zh-cn"] = "浩劫状况 2",
+		ru = "Условие Верной смерти 2",
 	},
 	label_mission_giver = {
 		en = "Mission Giver",
 		["zh-cn"] = "任务发布者",
+		ru = "Миссию дал:",
```

Comment:
additional colon?

Reply:
You can remove it, but it looks good this way too.
![screenshot_20241225_191515](https://github.com/user-attachments/assets/baf17b5d-08cf-4f0b-a48f-1acbee65fe20)